modules/geo-layers/src/mvt-layer/mvt-layer.js

```diff
@@ -15,7 +15,7 @@ const WORLD_SIZE = 512;
 const defaultProps = {
   uniqueIdProperty: {type: 'string', value: ''},
   highlightedFeatureId: null,
-  loaders: MVTLoader,
+  loaders: [MVTLoader],
```

Comment:
After this change, CartoBQTilerLayer throws:
```
tile-layer.js:18 Error: No valid loader found data: "", contentType: "application/vnd.mapbox-vector-tile" url: https://maps-api-v2.us.carto.com/user/public/bigquery/tileset/1/1/0
    at _callee$ (select-loader.js:29)
    at tryCatch (runtime.js:45)
    at Generator.invoke [as _invoke] (runtime.js:271)
    at Generator.prototype.<computed> [as next] (runtime.js:97)
    at asyncGeneratorStep (asyncToGenerator.js:3)
    at _next (asyncToGenerator.js:25)
    at asyncToGenerator.js:32
    at new Promise (<anonymous>)
    at asyncToGenerator.js:21
    at selectLoader (select-loader.js:13)
```

Taking a look to understand why and to fix it

Reply:
When `loaders` is a single loader, it is automatically selected. When it's an array, it's used as one of the candidates. Your endpoint gives no indication that it is a mvt loader compatible format.

Reply:
Looked at your error message again, maybe you are not using the latest loader patch?

Reply:
It fails using the scripting... It looks we need to bump @loaders.gl.  https://github.com/visgl/deck.gl/blob/master/yarn.lock#L1786

Reply:
PR added: https://github.com/visgl/deck.gl/pull/5477